### ISSUE TITLE
Fix issue with Colorizer configuration dependencies.

### ISSUE THREAD

**Issue by jacine** [apigee/apigee-devportal-kickstart-drupal#213]
**Describe the bug**
Configuration import validation fails.

**To Reproduce**
1. Run `drush cex -y` to export update config.
2. Run `drush cim -y` which should validate and tell you there are no pending changes.
3. See error with configuration validation:

```
Drupal\Core\Config\ConfigImporterException: There were errors validating the config synchronization.                              [error]
Configuration <em class="placeholder">customizer.theme.apigee_kickstart</em> depends on the <em
class="placeholder">customizer</em> extension that will not be installed after import. in
Drupal\Core\Config\ConfigImporter->validate() (line 737 of
/Users/jacine/Sites/kickstart_m10n/web/core/lib/Drupal/Core/Config/ConfigImporter.php).
The import failed due for the following reasons:                                                                                  [error]
Configuration <em class="placeholder">customizer.theme.apigee_kickstart</em> depends on the <em
class="placeholder">customizer</em> extension that will not be installed after import.
```

**Expected behavior**
Configuration should import successfully.

**Comment by shadcn** [comment]
I added a fix for this in #218

**Comment by jacine** [comment]
Fixed by #217.
